src/components/safe-apps/SafeAppIcon/index.tsx

```diff
@@ -1,17 +1,19 @@
 import { type ReactElement, memo } from 'react'
 
-const getIframeContent = (src: string): string => {
-  return `<style>
-    html, body {
-      padding: 0;
-      margin: 0;
-      background-color: transparent;
-      height: 100%;
-    }
-    body {
-      background: url('${src}') center center/contain no-repeat;
-    }
-  </style>`
+const APP_LOGO_FALLBACK_IMAGE = `/images/apps/app-placeholder.svg`
+
+const getIframeContent = (url: string, alt: string, width: number, height: number): string => {
+  return `
+     <body style="margin: 0;">
```

Comment:
Some browsers add margins to <html> too, please test in FF, Safari, etc.

Reply:
Tested in Safari desktop/mobile, chrome and FF - all good